qa/tasks/cephfs/test_nfs.py

```diff
@@ -21,6 +21,12 @@ def _cmd(self, *args):
     def _nfs_cmd(self, *args):
         return self._cmd("nfs", *args)
 
+    def _nfs_complete_cmd(self, cmd: str):
```

Comment:
Why is this helper required? How is it different than `nfs_cmd`?

Reply:
_nfs_cmd uses raw_cluster_cmd which only returnd stdout, while run_cluster_cmd (which this helper uses) returns process obj i.e. stdout,stderr and return code, when the cluster creation says cluster already exists, its fed into stderr not stdout thus making _nfs_cmd unusable in such scenarios